content/docs/components/serving/overview.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "Overview"
+description = "Model Serving Overview"
+weight = 1
++++
+
+Kubeflow provides two supported open source model serving systems that allow multi-framework model serving: KFServing and Seldon Core. You should choose the framework that best supports your model serving requirements. If you only serve  Tensorflow models using Tensorflow Serving is an option although both KFServing and Seldon support Tensorflow. A rough comparison between KFServing and Seldon Core is shown below:
```

Comment:
It took me a while to parse this part of the sentence: "If you only serve  Tensorflow models using Tensorflow Serving is an option...". At first I read "If you only serve  Tensorflow models using Tensorflow Serving" but then the rest of the sentence didn't make sense.

Recommendation: "If you only serve  TensorFlow models, TensorFlow Serving is an option..."

Reply:
I will remove it and create a separate section for TensorFlow Serving.